Description: Correct pod NAME issues in LedgerSMB/File/Transaction.pm
 Has the module name but not the dash plus short description in the NAME entry.
 Spelling error in line 9: trnasactions transactions.
Bug: http://sourceforge.net/tracker/?func=detail&aid=3472599&group_id=175965&atid=1005191
Author: Robert James Clay <jame@rocasa.us>

--- a/LedgerSMB/File/Transaction.pm
+++ b/LedgerSMB/File/Transaction.pm
@@ -2,11 +2,11 @@
 
 =head1 NAME
 
-LedgerSMB::File::Transaction
+LedgerSMB::File::Transaction - Manages attachments to financial transactions
 
 =head1 SYNPSIS
 
-Manages attachments to financial trnasactions (in 1.3, AR, AP, and GL entries)
+Manages attachments to financial transactions (in 1.3, AR, AP, and GL entries)
 
 =head1 INHERITS
 
